templates/zookeeper-env.sh.j2

```diff
@@ -0,0 +1,5 @@
+{% if zookeeper_env is defined %}
```

Comment:
good catch!  (and better solution to just uncommenting the variable altogether in the defaults (aka making it 'undefined')